src/main/rdf/vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL.ttl

```diff
@@ -11685,6 +11707,18 @@ quantitykind:RotaryShock
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "rotary shock"@en-US .
 
+quantitykind:RotationalFrequency
```

Comment:
How is this different from AngularVelocity that already exists?

Reply:
I don't know where that was coming from. I had already identified the qk I need as  `qudt:AngularFrequency`. Never mind. Will remove from PR.

Reply:
No, actually, It seems we need RotationalFrequency: https://en.wikipedia.org/wiki/Rotational_frequency - this description explains the distiction between it and AngularVelocity/AngularFrequency and maps nicely to http://standards.buildingsmart.org/IFC/DEV/IFC4_2/FINAL/HTML/link/ifcrotationalfrequencymeasure.htm

Reply:
... and I would also like to suggest new units for RPM and cycles per second (that's the one they use in IFC)

Reply:
AngularFrequency and AngularVelocity confuse me. 

The suggested distinction of RotationalFrequency and RotationalVelocity is quite subtle, but probably ok.

my suggestion about RPM was an error. I ended up finding REV-PER-MIN, and that one is fine.

Reply:
Looking at https://en.wikipedia.org/wiki/Rotational_frequency it seems that Hz, cycles per time and revolutions per time would all be units of rotational frequecy. I don't know what we would need rotational velocity for or what units it should be associated with.

Reply:
OK, I propose we make RotationalFrequency and RotationalVelocity qudt:exactMatch for now. We could refine this once we fully support vector datatypes. The applicable units would include Hertz, cycles per time, and revolutions per time.

Also, that we make AngularFrequency and AngularVelocity qudt:exactMatch.  The applicable units would include angular units per time (e.g. radians per second), but not Hertz.

Finally, I don't really see the Rotational and Angular quantity kinds with either family clearly skos:broader of the other, so I propose we simply cross reference them with rdfs:seeAlso.

Reply:
i'll do exactly that.

Reply:
done